esp/src/eclwatch/MemberOfWidget.js

```diff
@@ -62,9 +62,7 @@ define([
             this.grid.on("dgrid-datachange", function(event){
                 if (dojoConfig.isAdmin && params.username === context.currentUser && event.oldValue === true) {
                     var msg = confirm(context.i18n.RemoveUser + " " + event.rowId + ". " + context.i18n.ConfirmRemoval);
-                    if (!msg){
-                        event.preventDefault();
-                    }
+                    msg ? location = location.origin : event.preventDefault();
```

Comment:
How I had originally I'll modify. Location is just a way of refreshing the browser using Javascript. location.origin in this case is http://IP:PORTnumber